src/app/components/dashboard/dashboard.component.ts

```diff
@@ -3,35 +3,55 @@ import { Router } from '@angular/router';
 import { Data } from 'src/interfaces/dashboard';
 import { RestService } from 'src/app/services/rest.service';
 import { DailyQuote } from 'src/interfaces/daily-quote';
+import { FilterFolderPipe } from 'src/app/filter-folder.pipe';
 declare var $: any;
 
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
-  styleUrls: ['./dashboard.component.scss']
+  styleUrls: ['./dashboard.component.scss'],
+  providers: [FilterFolderPipe]
 })
 export class DashboardComponent implements OnInit {
 
   public quote: DailyQuote;
-  constructor(private route: Router, private apiService: RestService) { }
+
+  constructor(private route: Router, private apiService: RestService, private filterFolder: FilterFolderPipe) {}
+
   data: any;
   Username: string = null;
+  filterFolderName= '';
 
   ngOnInit() {
     this.gettingData();
     this.getQuote();
+    console.log(this.filterFolderName)
   }
   // GETTING USER DATA
   async gettingData() {
     const response = await this.apiService.getFoldersData();
     const data = await response.content;
+    console.log(data)
     this.data = data.folders;
```

Comment:
This won't work as the data is coming in the form of object. Instead, do something like

```
    data.folders.forEach((obj) => {
      this.data.push(obj);
    })
```
Also the variable `data` here will be of type array i.e `data:any[ ] = [ ]`